app/script/user/UserRepository.js

```diff
@@ -705,7 +705,7 @@ z.user.UserRepository = class UserRepository {
         return this.save_user(user_et, true);
       })
       .catch(error => {
-        this.logger.error(`Unable to load self user: ${error}`);
+        this.logger.error(`Unable to load self user: ${error}`, error);
```

Comment:
Seems like error was a string in this case. Adding it after the colon will cause the rest of the error message to disappear.